.github/workflows/ci.yml

```diff
@@ -220,6 +221,27 @@ jobs:
           make VERBOSE=1 -j${BUILD_CONCURRENCY} ${make_targets}
           ctest --timeout ${TEST_TIMEOUT} --output-on-failure ${ctest_flags}
 
+  dr-testing:
+    name: Run DR shp-tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build docker image
```

Comment:
Is a container really needed for these tests? The other oneDPL CI jobs run without it.

Reply:
Indeed not needed. Moved to linux-testing.